index.css

```diff
@@ -0,0 +1,96 @@
+.Main-Page {
+  max-width: 500px;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 590px;
+  width: 375px;
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  border: 2px solid black;
+  padding: 120px, 24px, 120px, 24px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: #3c3a39;
+  width: 375px;
+  height: 92px;
+}
+
+.section {
+  height: 590px;
+  width: 375px;
+  background: black;
+}
```

Comment:
- [ ] Please insert the background image in your design as is shown in the Figma template below.
> Figma
![image](https://user-images.githubusercontent.com/90556221/176877580-dff1243e-6db4-4fec-a005-06cb0c5e4909.png)

Currently your webpage has no background decorations